eo-runtime/src/main/java/org/eolang/BytesRaw.java

```diff
@@ -0,0 +1,252 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016-2024 Objectionary.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.eolang;
+
+import java.nio.ByteBuffer;
+import java.util.Arrays;
+
+/**
+ * Bytes to be created from byte array only.
+ *
+ * @since 1.0
+ * @todo #3239:90min Method {@link BytesOf#shift} should be refactored to get rid of
+ *  {@code @SuppressWarnings("PMD.CognitiveComplexity")} warning. You can
+ *  check description of this rules here
+ *  <a href="https://pmd.github.io/pmd/pmd_rules_java_design">pmd.github.io</a>
```

Comment:
@levBagryansky Since this todo now related only to `shift` method, you can move todo closer to it.

Reply:
@c71n93 I cannot, `Override` methods does not have commentary

Reply:
@levBagryansky understandable